app/javascript/availability/libraries_locations.json

```diff
@@ -138,22 +138,34 @@
     "DISABILITY": "Adaptive Technology and Services, 113W Pattee",
     "DISC-DN": "Dickinson Law (Carlisle) - CD/DVD Collection",
     "DISCOVERAB": "Abington - Discovery Zone",
-    "DISPLAY-AA": "Altoona - Display - 2nd Floor",
-    "DISPLAY-BD": "Behrend - On display",
-    "DISPLAY-BK": "Berks - Display",
-    "DISPLAY-BR": "Beaver - Display",
-    "DISPLAY-DN": "Dickinson Law (Carlisle) - Faculty Display",
-    "DISPLAY-DP": "Penn State Law (UP) - Faculty Display Case",
-    "DISPLAY-DS": "DuBois - Display",
-    "DISPLAY-ED": "Paterno - 5th Floor - Education Library Displays",
-    "DISPLAY-EG": "Engineering Library Display",
-    "DISPLAY-FE": "Fayette - New Titles or Featured Display Titles",
+    "DISPLAY-AA": "Altoona - On Display",
+    "DISPLAY-AB":	"Abington - On Display",
+    "DISPLAY-AC":	"Architcture - On Display",
```

Comment:
Should this be "Architecture"?

Reply:
Ahh good catch. I fixed that along with the weird white space some of them have (didn't show up that way in vs code)